src/utils/sqs.ts

```diff
@@ -0,0 +1,32 @@
+import { SqsService } from "./types";
+import {
+  SendMessageCommand, SendMessageCommandOutput,
+  SendMessageRequest,
+  SQSClient,
+} from "@aws-sdk/client-sqs";
+import { logger } from "./logger";
+
+export function sqsService(): SqsService {
+  const send = async function (messageBody: string): Promise<any> {
+    const { AWS_REGION } = process.env;
+    const { AUDIT_QUEUE_URL } = process.env;
+    const client = new SQSClient({ region: AWS_REGION });
+    const message: SendMessageRequest = {
+      QueueUrl: AUDIT_QUEUE_URL,
+      MessageBody: messageBody,
+    };
+
+    if (AUDIT_QUEUE_URL != null) {
+      try {
+        const result: SendMessageCommandOutput = await client.send(new SendMessageCommand(message));
+        logger.info(`Event sent with message id ${result.MessageId}`);
+      } catch (err) {
+        logger.error(`Failed to send message ${message.MessageBody} to SQS: ${err}`)
```

Comment:
Weird how sonarqube and `npm test:unit` report line 24 as uncovered despite the test being there.

Reply:
@alex9smith is this something you've seen before?

Reply:
It's not but I don't 100% trust Sonar's code coverage metrics - we've had a few instances where it's complained there's no tests for a file when there clearly are. I'm not sure about the lcov coverage though, I'll try and investigate

Reply:
Won't stop approval of this PR.